.travis.yml

```diff
@@ -78,15 +79,18 @@ before_script:
 
 script:
   - |
+    export RUN_DOWNLOAD_TEST=0
     # install gold standard and generate answers
     # this uses the yt gold standard version
-    export RUN_DOWNLOAD_TEST=0
-    export TRIDENT_GENERATE_TEST_RESULTS=1
-    git checkout $TRIDENT_GOLD
-    pip install -e .
-    pushd tests
-    py.test
-    popd
+    # answers are cached, so don't generate if they already exist
+    if [ ! -f $TRIDENT_ANSWER_DATA/test_results/TRIDENT_VERSION ]; then
```

Comment:
I guess I'm confused about the caching.  Won't there *always* be a file:

`$HOME/answer_test_data/test_results/TRIDENT_VERSION` ?

How will Travis know when we've updated to a new gold standard and it needs to regenerate the answers?